tests/Unit/Core/Search/Builder/TypedBuilder/ProductCombinationFiltersBuilderTest.php

```diff
@@ -59,7 +59,7 @@ public function testBuildFilters()
         $builder->setConfig(['request' => $this->buildRequestMock($productId, $shopId)]);
 
         $builtFilters = $builder->buildFilters();
-        $this->assertEquals(ProductCombinationFilters::generateFilterId($productId, $shopId), $builtFilters->getFilterId());
+        $this->assertEquals(ProductCombinationFilters::generateFilterId($productId), $builtFilters->getFilterId());
```

Comment:
It makes me realize we may have missed some unit tests for `ProductCombinationFilters::generateFilterId` even if the implementation evolved no tests to adapt so this part as probably forgotten we should add some simple unit tests